fastlane/Fastfile

```diff
@@ -1,5 +1,5 @@
 # Fastlane configuration file
-# 
+#
 # More information about multiple platforms in fastlane: https://github.com/KrauseFx/fastlane/blob/master/docs/Platforms.md
```

Comment:
Changes in this file should be reverted

Reply:
Readonly = false was true
this is the change that Houssem told me I must make.